File: curso-udemy/curso-algoritmo-completo/Módulo 10 - Java/Exercícios/exercicio_java/src/exercicio_java/EXCRETANGULO.java
package exercicio_java;
import java.util.Locale;
import java.util.Scanner;
import java.lang.Math;

public class EXCRETANGULO {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double base, altura, area, perimetro, diagonal;
		
		System.out.print("Base do Retângulo: ");
		base = sc.nextDouble();
		System.out.print("Altura do Retângulo: ");
		altura = sc.nextDouble();
		
		area = base * altura;
		perimetro = 2 * (base + altura);
		diagonal = Math.sqrt(Math.pow(base, 2.0) + Math.pow(altura, 2.0));
		
		System.out.println("Área = " + String.format("%.2f", area));
		System.out.println("Perímetro = " + String.format("%.2f", perimetro));
		System.out.println("Diagonal = " + String.format("%.2f", diagonal));
		
		sc.close();

	}

}

File: programming-logic/algorithm/knowing-c/Exercícios/EXCRETANGULO.c
#include <stdio.h>
#include <math.h>

int main(){

    double base, altura, areaTotal, perimetro, diagonal;

    printf("Base do retangulo: ");
    scanf("%lf", &base);
    printf("Altura do retangulo: ");
    scanf("%lf", &altura);

    areaTotal = (base * altura) / 2;
    perimetro = 2 * (base + altura);
    diagonal = sqrt(pow(base, 2.0) + pow(altura, 2.0));

    printf("\n");

    printf("Area : %.2lf \n", areaTotal);
    printf("Perimetro : %.2lf \n", perimetro);
    printf("Diagonal : %.2lf \n", diagonal);

    return 0;
}
